atlassian-terraform/terraform.tfvars

```diff
@@ -0,0 +1,93 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Sumo Logic, Jira Cloud, Jira Server (Onprem), OpsGenie, BitBucket credentials and installation options.
+
+# Sumologic
+sumo_access_id = "" # https://help.sumologic.com/Manage/Security/Access-Keys
+sumo_access_key = ""
+environment = "us1" # https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
+sumo_api_endpoint = "https://api.sumologic.com/api/v1/" # Make sure the trailing "/" is present. https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
+app_installation_folder = "Atlassian" # The Sumo Logic apps will be installed in a folder under your personal folder in Sumo Logic.
+
+# Sumo Logic Apps and Webhooks
+install_jira_cloud = "true"
+install_jira_on_prem = "true"
+install_bitbucket_cloud = "true"
+install_opsgenie = "true"
+# install_opsgenie should be true for the below option install_sumo_to_opsgenie_webhook to be true. https://help.sumologic.com/Manage/Connections-and-Integrations/Webhook-Connections/Webhook_Connection_for_Opsgenie
+install_sumo_to_opsgenie_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_opsgenie_webhook.json.tmpl for customizing payload.
+install_atlassian_app = "true"
+install_sumo_to_jiracloud_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_jiracloud_webhook.json.tmpl for customizing payload.
+install_sumo_to_jiraserver_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_jiraserver_webhook.json.tmpl for customizing payload.
+
+# Jira Cloud
+jira_cloud_url = "https://<example>.atlassian.net"
+jira_cloud_user = ""
```

Comment:
Is this mandatory? If so use something like: 
`jiar_cloud_user = "<cloud username>" # Provide your JIRA username`

Reply:
Added